nevergrad/functions/irrigation/irrigation.py

```diff
@@ -71,10 +73,8 @@ def __init__(self, symmetry: int, n_iterations: int = 1000) -> None:
         logging.info(f"we work on {self.cropname} with variety {self.cropvariety} in {self.address}.")
 
     def set_data(self, symmetry: int, k: int):
-        self.cropname = "rice"
-        self.cropvariety = np.random.RandomState(symmetry + 3 * k + 2).choice(
-            list(self.cropd.get_crops_varieties()[self.cropname])
-        )
+        self.cropname = "betterave"
```

Comment:
@Ethel2003 this won't work. 

Please replace by `self.cropname = "sugarbeet"`

Reply:
okay